include/spm.eu0.lst

```diff
@@ -75,7 +75,9 @@
 
 // fadedrv.c
 800677f8:fadeIsFinish
+800678e8:setTransition
```

Comment:
Could we call this fadeQueueTransition or something?

Reply:
This doesn't queue the transition though. All it does is set the values for which transitions will be used *when* a fadeout occurs.